static/showcases/multi-player.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8"/>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+  <title>Pillarbox Demo - Multi Player Showcase</title>
+  <link rel="icon" type="image/x-icon" href="../../img/favicon.ico">
+  <link rel="stylesheet" href="../../scss/index.scss"/>
+  <link rel="stylesheet" href="../../src/theme/theme.scss"/>
+  <style>
+
+    body {
+      max-width: var(--size-sm);
+    }
+
+    .showcase-content {
+      border: 1px solid var(--color-7);
+      border-radius: var(--radius-3);
+      padding: 0;
+      margin-top: var(--size-8);
+    }
+
+    .video-container {
+      width: 100%;
+      height: auto;
+      aspect-ratio: var(--ratio-widescreen);
+      padding: 0;
+    }
+
+    h2 {
+      padding-left: var(--size-2);
+    }
+
+    button {
+      width: 100%;
+      padding: var(--size-4) var(--size-3);
+      border: 0;
+    }
+
+    #close-btn {
+      border-radius: 0 0 var(--radius-3) var(--radius-3);
+    }
+  </style>
+  <!-- End Single Page Apps for GitHub Pages -->
+</head>
+
+<body>
+<div class="showcase-content">
+  <h2>Multiple players</h2>
+
+  <div class="video-container">
+    <video-js id="main-player" class="pillarbox-js" controls crossorigin="anonymous"></video-js>
+  </div>
+
+  <button id="toggle-player">Toggle main player</button>
+
+  <div class="video-container">
+    <video-js id="second-player" class="pillarbox-js" controls crossorigin="anonymous"></video-js>
+  </div>
+
+  <button id="close-btn">Close this window</button>
+</div>
+<script type="module">
+  import pillarbox from '@srgssr/pillarbox-web';
+
+  // Initialize the Main Player
+  const mainPlayer = pillarbox('main-player', { fill: true, debug: true });
+  mainPlayer.src({ src: 'urn:rts:video:6820736', type: 'srgssr/urn' });
+
+  // Initialize the Second Player
+  const secondPlayer = pillarbox('second-player', { fill: true, muted: true, debug: true });
+  // Tracking is disabled for this source, only one source can be tracked at a time.
+  secondPlayer.src({ src: 'urn:rts:video:6735513', type: 'srgssr/urn', disableTrackers: true });
+
+  // Add an event listener to a button with the id 'toggle-player'
+  document.querySelector('#toggle-player').addEventListener('click', () => {
+    // Toggle the mute state for both mainPlayer and secondPlayer
+    [mainPlayer, secondPlayer].forEach(player => {
```

Comment:
Why not `pillarbox.getAllPlayers`? it also demonstrates that a function exists for this use case

Reply:
Great suggestion, I've changed it.